docs/install/setting_up_cluster_on_amazon_web_services.md

```diff
@@ -322,20 +331,21 @@ Destroying a cluster permanently destroys all the data on the local instance sto
 ```pre
 ./snappy-ec2 -k my-ec2-key -i ~/my-ec2-key.pem destroy cluster-name
 ```
-This retains the security groups created for this cluster. To delete this, use it with `--delete-` group option.
+This also deletes the security groups created for this cluster.
 
 <a id="ec2startclustrezeppelin"></a>
 #### Starting Cluster with Apache Zeppelin
 
-Optionally, you can start an instance of Apache Zeppelin server with the cluster. [Apache Zeppelin](https://zeppelin.apache.org/) is a web-based notebook that enables interactive notebook. The Zeppelin server is launched on the same EC2 instance where the lead node is running.
+Optionally, you can start an instance of Apache Zeppelin server with the cluster. [Apache Zeppelin](https://zeppelin.apache.org/) provides a web-based interactive notebook that is pre-configured for you to talk to the SnappyData cluster. The Zeppelin server is launched on the same EC2 instance where the lead node is running.
```

Comment:
Suggestion: 
[Apache Zeppelin](https://zeppelin.apache.org/) provides a web-based interactive notebook that is pre-configured to communicate with SnappyData cluster.